libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -76,7 +71,6 @@ public class SalesforceAnalyticsManager {
 
     private static Map<String, SalesforceAnalyticsManager> INSTANCES;
     private static boolean sPublishHandlerActive;
-    private static ScheduledFuture sScheduler;
```

Comment:
There are a few touch points in this source, but its great that Android background tasks support cancelling existing work before enqueuing the same item.  That alleviates this Java code from that task.